python/lsst/ci/hsc/gen2/validate.py

```diff
@@ -174,19 +174,7 @@ def checkPsfStarsAndFlags(self, catalog, minStellarFraction=0.95, doCheckFlags=T
         """Utility function for derived classes that want to verify PSF source
         selection and flag setting
         """
```

Comment:
Help me understand what's going on here.  This would be so easier to talk about it in person, but instead I'll just ask a bunch of questions here so you can see how confused I am.  Again, the barrage of questions are not challenging what's here, they're just to show you how someone who didn't do the investigation could read this.